### PR TITLE
add Dokku

### PR DESCRIPTION
**Pull request by Cri5**

### CODE REVIEW

**Comment by igorshubovych** [comment]
I did not use Dokku really. But it LGTM.

**Comment by Cri5** [comment]
These are the most basic and most used commands, I didn't want to put too much examples as stated in contribution guide.

**Comment by igorshubovych** [comment]
Ok. Let's wait for others feedback.

**Comment by igorshubovych** [comment]
Could you please squash the commits?

File: pages/common/dokku.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Dokku
+
+> Docker powered mini-heroku
```
Comment (waldyrious) [inline pages/common/dokku.md]: "Docker-powered". Also, "mini-heroku" is not very self-explanatory; I would suggest expanding a little on what that means.

File: pages/common/dokku.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- List of runinng apps
```
Comment (waldyrious) [inline pages/common/dokku.md]: "List running apps", to match the style of the other example descriptions

**Comment by waldyrious** [comment]
Nice contribution, thanks! I added a couple inline comments.

**Comment by ostera** [comment]
@Cri5: Looks okay, needs rebasing/squashing.

**Comment by Cri5** [comment]
@ostera: Could you give me a tip how to do this, it's my first contribution to repo, I've only used git for my private projects until now.

**Comment by igorshubovych** [comment]
Read this
https://help.github.com/articles/about-git-rebase/

Do NOT pull from master.

You have 3 commits in PR. Run something like 
`git rebase --interactive HEAD~3`

Then edit rebase message file:
Pick for "Add Dokku" commit
Squash or fixup for "Update Dokku" commit
Remove Merge commit

Check your log with `git log`. If you have only a single commit for this change then you can do

`git push --force`

P.S. for future it is better to use feature branches

**Comment by Cri5** [comment]
@igorshubovych: Thanks for help, it's done.

**Comment by waldyrious** [comment]
Are the changes to pip intended to be in this PR?

**Comment by Cri5** [comment]
@waldyrious: No, damn, I messed something. I think I squashed one commit too much. I only added dokku.md

**Comment by igorshubovych** [comment]
You had to remove merge commit.

**Comment by igorshubovych** [comment]
http://superuser.com/questions/229290/how-to-amend-the-last-commit-to-un-add-a-file

**Comment by igorshubovych** [comment]
Thank you for implementation. I committed just fixed dokku.md.

**Pull request closed without merging**